laboratory/views/reports.py

```diff
@@ -16,13 +16,104 @@
 from weasyprint import HTML
 from django.utils.translation import ugettext as _
 
-from laboratory.models import Laboratory, LaboratoryRoom, Object, Furniture, ShelfObject, CLInventory
+from laboratory.models import Laboratory, LaboratoryRoom, Object, Furniture, ShelfObject, CLInventory, OrganizationStructure, PrincipalTechnician
 from laboratory.views.djgeneric import ListView
 from laboratory.decorators import user_lab_perms
 import django_excel
 from django.db.models.aggregates import Sum, Min
 
 
+def make_book_organization_laboratory(objects):
+    dev=[
+        [_("Name"), _("Phone"), _("Location"),_("Principal Technician"),_('Phones'),_('Emails')]
+        ]
+    for object in objects:
+        principaltechnician=""
+        phones=""
+        emails=""
+        for ptechnician in object.principaltechnician_set.all():
+            if principaltechnician != "" :
+                principaltechnician+=" \n"
+                phones+=" \n"
+                emails+=" \n"
+            principaltechnician += ptechnician.name
+            phones += ptechnician.phone_number
+            emails += ptechnician.email    
+        dev.append([
+            object.name,
+            object.phone_number,
+            object.location,
+            principaltechnician,
+            phones,
+            emails,
+            ])
+    return dev
+    
+    
+@login_required
+@user_lab_perms(perm="report")
+def report_organization_building(request, *args, **kwargs):
+    var = request.GET.get('organization')
+    if var:
+            org= get_object_or_404(
```

Comment:
If user allowed to see this organization ? 
Be worry about if organization is child of one user organization higher owned by the user